packages/cloudflare/src/api/kvCache.ts

```diff
@@ -2,7 +2,7 @@ import type { KVNamespace } from "@cloudflare/workers-types";
 import type { CacheValue, IncrementalCache, WithLastModified } from "@opennextjs/aws/types/overrides";
 import { IgnorableError, RecoverableError } from "@opennextjs/aws/utils/error.js";
 
-import { getCloudflareContext } from "./get-cloudflare-context.js";
+import { getCloudflareContext } from "./cloudflare-context.js";
```

Comment:
in this file you can remove the following pattern:

```ts
    if (!this.initialized) {
      await this.init();
    }
```

That was only required because `getCloudflareContext` was async

Reply:
I'll amend the PR

Reply:
ah ok I see what you meant, thanks for the update 👍